docs/Recipes.md

```diff
@@ -95,3 +95,31 @@ module.exports = ({ platform }, { module, resolve }) => ({
   },
 });
 ```
+
+## Detox
```

Comment:
This should rather be
```
## Detox with `react-native-repackager` mocks
```
You can use Detox without this package. We can adjust this title later, when we have more Detox tips :)

Reply:
How about ` ## Mock files when running detox tests` ?

Reply:
Will do